xgi/algorithms/connected.py

```diff
@@ -42,17 +51,19 @@ def is_connected(H):
     True
 
     """
-    return len(_plain_bfs(H, list(H.nodes)[0])) == len(H)
+    return len(_plain_bfs(H, list(H.nodes)[0], s=s)) == len(H)
 
 
-def connected_components(H):
+def connected_components(H, s=1):
     """
     A function to find the connected components of a hypergraph.
 
     Parameters
     ----------
     H: Hypergraph object
         The hypergraph of interest
+    s: int, optional
+        The overlap parameter
```

Comment:
describe more specifically, here or in general description of the function, like "two nodes are s-connected if they share at least s ..."

Reply:
Done.